### ISSUE TITLE
Testing version "hakuneko-desktop_4.1.4_linux_amd64.rpm" requires libc6 which seems to be unfindable on fedora 30

### ISSUE THREAD

**Issue by RMPR** [manga-download/hakuneko#264]
As stated in the title, I tried to install the testing version to report bugs and support the project at least in that way but I get an error message :
```
Error: 
 Problem: conflicting requests
  - nothing provides libc6 needed by hakuneko-desktop-4.1.4-0.x86_64
(try to add '--skip-broken' to skip uninstallable packages)
```
After fiddling a little bit : installing the latest glibc and compat-libstdc packages, ...  the error persists and I am only able to find a suitable package for debian or debian based distros.
Side notes : 
- the previous stable version 4.0.0 install is just fine not requiring (at least explicitly) any dependencies.
- ```sudo dnf provides libc6``` give zero output and ```sudo dnf provides "*libc6*"``` ask to install compat-libstdc++ (back to one of the solutions tried).

**Comment by 27136773** [comment]
The test version you used is an old one and was already [confirmed to not work](https://github.com/manga-download/hakuneko/issues/51#issuecomment-486452545).
The new test candidates are not published yet until the internal tests are complete.
See [Next Steps](https://github.com/manga-download/hakuneko/issues/51#issuecomment-511158897).

If everything works as expected, the test candidates will be available tomorrow.

Your feedback is still valuable, so i leave the ticket open until it is confirmed that the 5.0.6 test candidates are working.

**Comment by 27136773** [comment]
The same problem can also be reproduced with the test candidate 5.0.6
The HakuNeko RPM package is not build on Fedora and therefore set the dependencies based on the build system.
Actually it adds a dependency to `libc6`, which is not required for Fedora (glibc).
Furthermore HakuNeko requires `libXss.so.1` to run on Fedora, which resolves to the dependency of `libXScrnSaver`

Possible Solution:

- Build RPM package on a Fedora agent
- Set custom RPM dependencies instead of autodetecting the dependencies on the build system

**Comment by 27136773** [comment]
New test candidates are now available, see #266
